extensions/amp-image-viewer/0.1/validator-amp-image-viewer.protoascii 

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-image-viewer
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-image-viewer extension .js script"
+  requires: "amp-image-viewer"
+  extension_spec: {
+    name: "image-viewer"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-image-viewer>
+  html_format: AMP
+  tag_name: "AMP-IMAGE-VIEWER"
+  satisfies: "amp-image-viewer"
+  requires: "amp-image-viewer extension .js script"
+  attr_lists: "extended-amp-global"
+  spec_url: ""
```

Comment:
please remove if not known, usually it's of the form `https://www.ampproject.org/docs/reference/components/amp-image-viewer`